doc/whats_new/v0.23.rst

```diff
@@ -146,6 +146,14 @@ Changelog
   applies the correct inverse transform to the transformed data. :pr:`16655`
   by :user:`Lewis Ball <lrjball>`.
 
+- |API| The default value of the :code:`whiten` argument in
```

Comment:
This is not what is implemented in this PR. This PR implements something more like ```|Enhancement| the `whiten` parameter to :class:`decomposition.FastICA` may now be set to 'unit-variance'```...